### PR TITLE
added donation banner

### PR DESCRIPTION
**Pull request by Stefterv**

### CODE REVIEW

**Comment by Stefterv** [comment]
preview https://p5-website.steftervel.de/#previewdonationbanner

**Comment by SableRaf** [comment]
@davepagurek

File: src/layouts/BaseLayout.astro [hunk 1]
```diff
@@ -84,6 +84,12 @@ const headerTopic = topic
       const htmlElement = document.documentElement;
       htmlElement.className = `${storedSettings.join(" ")} ${htmlElement.className}`;
     </script>
+    <link
+      rel="stylesheet"
+      href="https://foundation-donate-banner.netlify.app/static/css/main.css"
```
Comment (limzykenneth) [inline src/layouts/BaseLayout.astro:89]: @Stefterv @SableRaf The CSS here is overwriting the CSS variable used on the website to render monospace font `--font-serif`. Can the variable be scoped or if it is not used at all be removed from the CSS served?

Comment (limzykenneth) [reply src/layouts/BaseLayout.astro:89]: Just saw that this was already mentioned in #649

Comment (Stefterv) [reply src/layouts/BaseLayout.astro:89]: Hey Kenneth! Thank you for the heads up, we've pushed a fix to the banner repo to fix this

**Pull request merged**